src/blob.rs

```diff
@@ -377,16 +378,27 @@ impl From<Hash> for iroh::bytes::Hash {
     }
 }
 
+/// The `progress` method will be called for each `AddProgress` event that is
+/// emitted during a `node.blobs_add_from_path`. Use the `AddProgress.type()`
+/// method to check the `AddProgressType`
```

Comment:
if you want to be really nitpciky, both rust's and python's doc comments take the first paragraph as a summary to be shown in indexes etc.  So it common to make sure it is a single sentence which fits on one line and separate them with a blank line from the rest of the docs.

Reply:
great point, thank you!